front/app/components/HookForm/Feedback/index.tsx

```diff
@@ -109,6 +114,7 @@ const Feedback = ({ successMessage, onlyShowErrors }: FeedbackProps) => {
           {errorMessageIsShown && (
             <Error
               marginBottom="12px"
+              {...errorProps}
               text={
```

Comment:
(Allows same styling options for our hook Feedback component by including the ErrorProps)